tests/unit/test_filterout_utils.py

```diff
@@ -116,17 +117,100 @@ def test_delete_conflicts_more_deletion():
     assert len(filter_conflicts_by_path(conflicts, 'figures')) is 1
 
 
-def test_filter_out():
+def test_filter_conflicts():
```

Comment:
Didn't you add an extra test here? For keys that are substrings? (I know that is currently fixed in this implementation, but I think it's nice to have for future refactors)

Reply:
I didn't, @ammirate suggested one in https://github.com/inspirehep/inspire-json-merger/pull/40#discussion_r148733379 and I've just added it.

Reply:
👍 I missed it